nutils/solver.py

```diff
@@ -403,15 +403,15 @@ class newton(cache.Recursion, length=1):
     '''
 
     @types.apply_annotations
-    def __init__(self, target, residual: integraltuple, jacobian: integraltuple = None, lhs0: types.arraydata = None, relax0: float = 1., constrain: arrayordict = None, linesearch=None, failrelax: types.strictfloat = 1e-6, arguments: argdict = {}, **kwargs):
+    def __init__(self, target, residual: integraltuple, jacobian: integraltuple = None, lhs0: types.arraydata = None, relax0: float = 1., constrain: arrayordict = None, linesearch='__legacy__', failrelax: types.strictfloat = 1e-6, arguments: argdict = {}, **kwargs):
```

Comment:
Suggestion: add a unittest for Newton with disabled line search (e.g. `test.test_solver.{navierstokes,finitestrain}.test_newton_nolinesearch`).

Reply:
Done - this time also for finitestrain which I had initially missed. Could you re-approve?